pkg/gofr/datasource/pubsub/kafka/kafka.go

```diff
@@ -92,6 +101,18 @@ func validateConfigs(conf Config) error {
 		return errBrokerNotProvided
 	}
 
+	if conf.BatchSize <= 0 {
+		return errBatchSize
+	}
+
+	if conf.BatchBytes <= 0 {
+		return errBatchBytes
+	}
+
+	if conf.BatchTimeout <= 0 {
+		return errBatchTimeout
+	}
```

Comment:
rather than returning an error we can give a warn log and use default value incase of any invalid value

Reply:
I would keep errors in case of wrong value, it is better to follow the principle to fail application start rather then replace values set by user with defaults.